SciPy2024/module2/embeddings.ipynb

```diff
@@ -0,0 +1,376 @@
+{
```

Comment:
<p>Typo found for "paragraph" in Case 1. Also typo for "Tokenizing" for both Case 1 and Case2.</p><p> </p><p>Also, Question: why is there a shape of 5 for the vector for the entire paragraph (Case 1), where I see only 4 blocks of sentences?</p><ol><li>Earth is a planet of the solar system</li><li>There are 9 planets in the solar system</li><li>All planets revolve around the sun</li><li>Sun is a start</li></ol>
<br/>

 _Reply via <a href='https://app.reviewnb.com/uw-ssec/tutorials/pull/34/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='6'/>

Reply:
<p>@lsetiawan Thanks for highlighting the typos.</p><p> </p><p>Exactly the right question that I wanted to get asked!  </p><p> </p><p>So the shape of the vector is based on the dimension of the embedding model. </p><p>Example: 'apple' can be converted to a vector embedding of N=768 dimensions for word based embedding. [2.2, 3.5, 4.5, ...... total 768 length]</p><p> </p><p>Similarly, a paragraph/document can be converted into a vector embedding of N = 5 (or whatever dimension the model supports). This has no relation with the number of sentences in the paragraph.</p>

Reply:
Got it. okay I didn't know that! Makes sense. Thanks